README.md

```diff
@@ -8,6 +9,7 @@ Missing icon
 ### [CrashLoopBackOff Rescheduler](pkg/remediator/crash_loop_back_off_rescheduler.go)
 
 Reschedules Pods in CrashLoopBackOff.
+- Runs every X = `frequency` (configurable).
```

Comment:
should be `every 1m (configurable)` to be symmetric with the restartCount ... or use `> X` for restartCount